lib/auto-trace.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
```

Comment:
ah we forgot to put the build files into the .gitignore. It will make pull requests a lot cleaner in the future if we do so. Here are the steps:

1) add `lib` as a new line in the .gitignore
2) create a `.npmignore` file. You can either leave it empty or you can put a comment like `# empty so that lib is gitignored but not npmignored`
3) `git rm --cached lib` should delete the files from version control